tests/Feature/Generator/MigrationGeneratorTest.php

```diff
@@ -149,9 +150,17 @@ public function output_also_creates_pivot_table_migration()
         $this->files->expects('put')
             ->with($model_migration, $this->fixture('migrations/belongs-to-many.php'));
 
+        Config::set('blueprint.use_constraints', false);
```

Comment:
Seems odd that this is set and unset in the same test. I also think it needs to be through `$this->app->config` for Testbench.

Reply:
This decision was made to test the migration when `use_constraints` is enabled and disabled.

Reply:
It would be good to see these in two separate tests. In fact, I would expect you only need one test to demonstration the constraints.

(and another to demonstrate using `foreignId` for Laravel 7+.

Reply:
I still don't understand this line. Why is it needed when it's defaulted to `false` in `TestCase::setUp`?